packages/react-components/src/atoms/MultiSelect.tsx

```diff
@@ -20,27 +20,36 @@ const containerStyles = css({
   flexBasis: '100%',
 });
 
+export type MultiSelectOptionsType = {
+  isFixed?: boolean;
+  label: string;
+  value: string;
+};
+
 type RefType =
-  | Select<OptionTypeBase, true>
-  | AsyncSelect<OptionTypeBase, true>
+  | Select<MultiSelectOptionsType, true>
+  | AsyncSelect<MultiSelectOptionsType, true>
   | null;
 
 type MultiSelectProps = {
   readonly customValidationMessage?: string;
   readonly id?: string;
   readonly enabled?: boolean;
   readonly placeholder?: string;
-  readonly onChange?: (newValues: OptionsType<OptionTypeBase>) => void;
-  readonly values?: OptionsType<OptionTypeBase>;
+  readonly onChange?: (newValues: OptionsType<MultiSelectOptionsType>) => void;
```

Comment:
Personally, I would prefer to expose `onChange` and `components` and let clients overwrite the behaviour they want, instead of adding the fixed behaviour to all intances of this component.